x-pack/plugins/snapshot_restore/public/application/components/restore_snapshot_form/steps/step_logistics/step_logistics.tsx

```diff
@@ -573,7 +600,7 @@ export const RestoreSnapshotStepLogistics: React.FunctionComponent<StepProps> =
             <FormattedMessage
               id="xpack.snapshotRestore.restoreForm.stepLogistics.includeGlobalStateDescription"
               defaultMessage="Restores templates that don’t currently exist in the cluster and overrides
-              templates with the same name. Also restores persistent settings and all system indices. {learnMoreLink}"
+              templates with the same name. Also restores persistent settings and all system indices from all features. {learnMoreLink}"
```

Comment:
Restore the global cluster state and the configuration, history, and data used by features such as Elasticsearch security.